src/app/(proper_react)/(redesign)/(authenticated)/user/welcome/EnterInfo.tsx

```diff
@@ -46,7 +49,8 @@ const createProfileAndStartScan = async (
 
   const result: WelcomeScanBody = await response.json();
   if (!result?.success) {
-    throw new Error("Could not start scan");
+    logger.error("Could not start scan", { response });
```

Comment:
Hm, I'm surprised that this works, `EnterInfo` is a client component right and `winston` is server (Node) right? I think it'd be better to use `Sentry.captureException`, you can get that with `import Sentry from "@sentry/nextjs"` directly in the component then you don't need to pass it as a prop.

I'm curious now if passing a server-side function as a prop like this does work, I *think* that props being passed from React server to client worked via [structured cloning](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm) or similar.

Alternatively a server action could be used here, but I would just use Sentry it's much simpler and safer than trying to do it ourselves.

Reply:
Just checked and https://react.dev/reference/rsc/use-server#serializable-parameters-and-return-values are the rules for what can be passed from server to client components.

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4725/commits/e69a817bbc8f2c66908fb3bfc0aa8ea268686e1a

Reply:
The above commit created breaking changes which are fixed here:
https://github.com/mozilla/blurts-server/pull/4725/commits/9b692572c67661956945f6b2b4cfe2c5b1634b21

Reply:
Updated to use NextJS import of Sentry instead of node.  Reverted changes in previous commit.
https://github.com/mozilla/blurts-server/pull/4725/commits/4861861de8badf0e92d0c812305dfe9cf5fa3992